src/test/cypress/integration/exercises/text/TextExerciseManagement.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { BASE_API } from '../../../support/constants';
+import { DELETE } from '../../../support/constants';
+import { generateUUID } from '../../../support/utils';
+import { artemis } from '../../../support/ArtemisTesting';
+import dayjs from 'dayjs';
+
+// The user management object
+const users = artemis.users;
+
+// Requests
+const courseManagement = artemis.requests.courseManagement;
+
+// PageObjects
+const textCreation = artemis.pageobjects.textExercise.creation;
+const exampleSubmissions = artemis.pageobjects.textExercise.exampleSubmissions;
+const exampleSubmissionCreation = artemis.pageobjects.textExercise.exampleSubmissionCreation;
+const navigationBar = artemis.pageobjects.navigationBar;
+const courseManagementPage = artemis.pageobjects.courseManagement;
+const courseManagementExercises = artemis.pageobjects.courseManagementExercises;
+
+describe('Text exercise management', () => {
+    let course: any;
+
+    before(() => {
+        cy.login(users.getAdmin());
+        courseManagement.createCourse().then((response) => {
+            course = response.body;
+        });
+    });
+
+    it('Creates a text exercise in the UI', () => {
+        cy.visit('/');
+        navigationBar.openCourseManagement();
+        courseManagementPage.openExercisesOfCourse(course.title, course.shortName);
+        cy.get('[jhitranslate="artemisApp.textExercise.home.createLabel"]').click();
+
+        // Fill out text exercise form
+        const exerciseTitle = 'text exercise' + generateUUID();
+        textCreation.typeTitle(exerciseTitle);
+        textCreation.setReleaseDate(dayjs());
+        textCreation.setDueDate(dayjs().add(1, 'days'));
+        textCreation.setAssessmentDueDate(dayjs().add(2, 'days'));
+        textCreation.typeMaxPoints(10);
+        textCreation.checkAutomaticAssessmentSuggestions();
+        const problemStatement = 'This is a problem statement';
+        const exampleSolution = 'E = mc^2';
+        textCreation.typeProblemStatement(problemStatement);
+        textCreation.typeExampleSolution(exampleSolution);
+        textCreation.create().its('response.statusCode').should('eq', 201);
```

Comment:
```suggestion
        textCreation.create();
```
Unless you want to check that the status code is *exactly* 201 this assertion is redundant since non 2xx codes fail the test anyways.

Reply:
As far as I know cypress only enforces these status codes on requests sent directly from cypress. Since this is an intercepted the validation does not apply here, so I have to verify it explicitly